CHANGELOG.md

```diff
@@ -1,12 +1,208 @@
 # Changelog
 
+<a name="1.3.0"></a>
+## 1.3.0 (2020-02-13)
+
+### Added
+
+- ✅ : wait for the job title to appear [[d9b52c5](https://github.com/CodeKaio/gaia/commit/d9b52c52cb70718a4e5c1c4d9e70afd3309e76e3)]
```

Comment:
links are still targeting the old repository url `https://github.com/CodeKaio/gaia`.

Is it normal ?

Reply:
Oh, I think it took my git remote, which is the old one, I'll do the replace